cfme/fixtures/tag.py

```diff
@@ -13,9 +14,13 @@ def category():
         Returns random created category object
         Object can be used in all test run session
     """
+    if BZ(1517285, forced_streams='5.9').blocks:
+        display_name = 'test-{}'.format(fauxfactory.gen_alphanumeric(length=32))
```

Comment:
**Required**
Drop the length passed to aphanumeric from `32` to `27` to account for the characters in 'test-' and in both cases use a 32 character name for testing.

Reply:
I would go a step further and make that number 27 be calculated, and leave a comment about the field it is going into being a max length of 32.  This makes the number 27 completely unmysterious, and answers any questions someone reading the code would have.